packages/lib-classifier/src/plugins/tasks/SimpleDropdownTask/models/SimpleDropdownAnnotation.js

```diff
@@ -0,0 +1,18 @@
+import { types } from 'mobx-state-tree'
+import Annotation from '../../models/Annotation'
+
+const SimpleDropdown = types.model('SimpleDropdown', {
+  value: types.optional(types.frozen({
+    value: types.string,  // Value selected by the user
```

Comment:
The readme notes that this is the index of the value from the options array, so this should be `types.number`

Reply:
DERP you're right! String is incorrect; I should have converted this to `number` when I copied over the code